docker/package/tezos_setup_wizard.py

```diff
@@ -578,6 +595,13 @@ def import_snapshot(self):
             snapshot_file = TMP_SNAPSHOT_LOCATION
 
             if self.config["snapshot"] == "skip":
```

Comment:
I think it makes sense to ask for the desired history mode even if the user decided to download a snapshot or provide one stored locally. What do you think about providing the same choice when the user downloaded a full snapshot (well, except for the fact, that there are going to be only two options: full and archive)?

This way we may combine these changes and changes from the #365 and the history mode choice will be more consistent within various bootstrap options

Reply:
Also, there is a possibility to run a rolling node using a full snapshot, which is probably not useful, but in some cases, some snapshot providers provide only full snapshots and thus it still may be useful

Reply:
Ah, if it's possible to start a rolling node from a full snapshot, it definitely makes sense to me to ask about the mode when importing

Reply:
Related to this: what do you think about inverting the order of the "syncing" and "history mode" choices?

If we ask for the history mode first then in the syncing question we can only show:
- "download rolling" if the chosen history was rolling
- "download full" otherwise (and use the option from #365 if archive was picked)

in either case this first option would be the default one (followed by the common "file"/"url"/"skip" ones).

Reply:
:thinking: I see a potential problem in not offering the `download full` snapshots option if a `rolling` history mode was chosen, related to the whole "snapshot providers only providing full snapshots", but I guess it's a smaller subset of users that are going to run into this compared to the potential UX improvement, so I agree